AlbumDownloader.py

```diff
@@ -227,7 +232,7 @@ def choose_video(soup, song_title, artist, wanted_length=None, num_of_choices=3)
     return items_hrefs[score.index(max(score[::-1]))]
 
 
-def download_song(song_title, artist, wanted_length=None, output_dir=r"C:\Users\User\Music"):
+def download_song(song_title, artist, wanted_length=None, output_dir=r"C:\Users\talsi\Desktop\music from tomer"):
```

Comment:
change it back. and please open an issue to check the users real name and put it in his own Music folder